bin/generate-metadata-yaml

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+set -euf
```

Comment:
Oh these are actually my standard options since somebody passed me [this](https://sipb.mit.edu/doc/safe-shell/) link in a review. Personally I supported that because I:

- usually want my script to stop if something went wrong even when I forgot explicit error handling (`-e`)
- expect that a not explicitly set variable is most likely a typo or a similar bug (`-u`)
- did not use globbing and was afraid of unexpected magic (`-f`)

But thank you for four view on this topic. Will think more in detail about this next time.

For this code it is very simple and obvious: It should follow the conventions of the other scripts in the repository :)